domain/metadata/token_metadata.py

```diff
@@ -0,0 +1,76 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import Optional
+
+from domain.metadata.metadata import Metadata, MetadataType
+
+
+class TokenNFTType(str, Enum):
+    VIDEO = 'VIDEO'
+    IMAGE = 'IMAGE'
+    AUDIO = 'AUDIO'
+
+
+@dataclass
+class TokenNFT:
+    """Data of token NFT
+
+    :param type: NFT Type. image or video.
+    :type type: :py:class:`domain.metadata.token_metdata.TokenNFTType`
+
+    :param file: Media file of the NFT
+    :type file: str
+    """
+    type: TokenNFTType
+    file: str
+
+
+@dataclass
+class MetaToken:
+    """Metadata of a token
+
+    :param id: Token unique id
+    :type id: str
+
+    :param verified: If token is verified or not. None and False are equivalent
+    :type verified: Optional[bool]
+
+    :param banned: If token is banned or not. None and False are equivalent
+    :type banned: Optional[bool]
+
+    :param reason: Bannishment reason
+    :type reason: Optional[str]
+
+    :param nft: NFT data, if any.
+    :type nft: Optional[:py:class:`domain.metadata.token_metdata.TokenNFT`]
```

Comment:
I was thinking about my task nowadays. I need to identify a NFT token even if it's not to show the digital asset on the explorer, i.e. the metadata of the token must have a way for me to know that it's a NFT without having this `TokenNFT` attribute set. 

What's the best way to achieve it? Having a boolean attribute `is_nft` and change this parameter to `nft_data`? Or maybe create a `type` that nowadays can be `NFT` or `NORMAL/DEFAULT`?

Reply:
The NFT you are talking about is not to be shown in explorer but still have a file, right? Even if not, we still can use `type` to achieve that. We only have `AUDIO`, `VIDEO` and `IMAGE` today, and only these are shown on explorer. We can add something like `OTHER` or other more specific type and it will just not be shown on explorer.